BLSQ/iaso
# POLIO-925: fix display good date format + no geojson

Date of onset is now displayed in format `DD/MM/YYYY` as the other date fields
Geojson show clear message if there is now Scope

https://user-images.githubusercontent.com/25134301/229091007-abe6ea0d-7afd-4b2b-b731-7ad8f47a378a.mov


Related JIRA tickets : [POLIO-925](https://bluesquare.atlassian.net/browse/POLIO-925?atlOrigin=eyJpIjoiYmUwOGRjZWYyMmNlNGQ4NjgxYTM4ZjQ4OTIwM2I2NmQiLCJwIjoiaiJ9)

## Self proofreading checklist


## Changes

- pass `getLogValue` to onset_at and creation_email_send_at to convert date and display it in good format
- show explicit message if no geographic data for geojon instead of displaying empty placeholder

## How to test

- Go to Campaign
- Click on a campaign 
- Add/Change value of **Date of onset** field in Base Info Tab, check if date is displayed in format DD/MM/YYYY
- Check if there is no scope, Geojson value should be "Without geographic data"
 

## Print screen / video

https://user-images.githubusercontent.com/25134301/229087584-b445f883-27b1-4ce0-96ac-67a9a5c178b1.mov

[POLIO-925]: https://bluesquare.atlassian.net/browse/POLIO-925?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
